net-misc/xmrig/metadata.xml

```diff
@@ -24,6 +24,12 @@
     <flag name="hwloc">
       Use <pkg>sys-apps/hwloc</pkg> for CPU affinity support
     </flag>
+    <flag name="cuda">
```

Comment:
Please keep in alphabetical order, e.g. `cuda`, `donate`, `hwloc`, `opencl`.

Reply:
I'll check `cuda`, give me a little bit of time 🙂

Reply:
I'm not sure why, but `libxmrig-cuda.so` isn't being built (or linked) even though the sources themselves are compiled. I've taken a look at the cmake files and am a little stumped.

`* CUDA         disabled (/usr/bin/libxmrig-cuda.so: cannot open shared object file: No such file or directory)`

Full build/install/find log: https://pastebin.com/raw/yVM5PGyX

Any ideas?